instrumentation/jsonrpc4j-1.3/library/src/main/java/io/opentelemetry/instrumentation/jsonrpc4j/v1_3/JsonRpcServerRequest.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.jsonrpc4j.v1_3;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.lang.reflect.Method;
+import java.util.List;
+
+public final class JsonRpcServerRequest {
```

Comment:
does this class need to be part of the public api of the module?

Reply:
removed `public`